code/game/objects/items.dm

```diff
@@ -131,6 +131,9 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 	/// Holder var for the item outline filter, null when no outline filter on the item.
 	var/outline_filter
 
+	/// A linked implant to teleport to
+	var/obj/item/organ/internal/cyberimp/linked_implant
```

Comment:
I am not sure adding a var on all items for implants is a good idea

Reply:
removed